### PR TITLE
resource/alicloud_ess_scaling_group: Add support for new parameter `tags`

### PR DESCRIPTION
**Pull request by TalentNing**
TF_ACC=1 go test ./alicloud -v -run=TestAccAlicloudEssScalingGroup -timeout=300m
=== RUN   TestAccAlicloudEssScalingGroupsDataSource
--- PASS: TestAccAlicloudEssScalingGroupsDataSource (79.53s)
=== RUN   TestAccAlicloudEssScalingGroup_basic
--- PASS: TestAccAlicloudEssScalingGroup_basic (114.51s)
=== RUN   TestAccAlicloudEssScalingGroup_launchTemplateIdAndTags
--- PASS: TestAccAlicloudEssScalingGroup_launchTemplateIdAndTags (58.19s)
=== RUN   TestAccAlicloudEssScalingGroup_costoptimized
--- PASS: TestAccAlicloudEssScalingGroup_costoptimized (92.29s)
=== RUN   TestAccAlicloudEssScalingGroup_vpc
--- PASS: TestAccAlicloudEssScalingGroup_vpc (91.56s)
=== RUN   TestAccAlicloudEssScalingGroup_slb
--- PASS: TestAccAlicloudEssScalingGroup_slb (127.41s)
PASS
ok      github.com/aliyun/terraform-provider-alicloud/alicloud  566.043s

### CODE REVIEW

File: alicloud/resource_alicloud_ess_scaling_group_test.go [hunk 1]
```diff
@@ -209,7 +209,7 @@ func TestAccAlicloudEssScalingGroup_basic(t *testing.T) {
 
 }
 
-func TestAccAlicloudEssScalingGroup_withLaunchTemplateId(t *testing.T) {
+func TestAccAlicloudEssScalingGroup_launchTemplateIdAndTags(t *testing.T) {
```
Comment (xinmu0426) [inline alicloud/resource_alicloud_ess_scaling_group_test.go:212]: 新增加一个测试用例吧，不要修改老的测试用例

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
 ## 1.160.0 (Unreleased)
+
+- resource/alicloud_ess_scaling_group: Add support for new parameter `tags`
+- datasource/alicloud_ess_scaling_group: Add support for new parameter `tags`
```
[the remark this answers is not in the document]
Comment (xinmu0426) [reply CHANGELOG.md:4]: commit 合并为1次

[the remark this answers is not in the document]
Comment (TalentNing) [reply CHANGELOG.md:4]: done

File: alicloud/resource_alicloud_ess_scaling_group_test.go (hunk 1 above)
Comment (TalentNing) [reply alicloud/resource_alicloud_ess_scaling_group_test.go]: done

**Pull request merged**